src/app/shared/components/notifications/notifications.component.html

```diff
@@ -1,22 +1,26 @@
 <ng-container *ngIf="!(isMobileScreen$ | async); else isMobileScreen">
   <button mat-icon-button [matMenuTriggerFor]="menu">
-    <mat-icon>notifications_none</mat-icon>
+    <mat-icon class="notifications-menu" [matBadge]="notificationsAmount?.amount || null">
+      notifications_none
+    </mat-icon>
   </button>
-  <span *ngIf="notificationsAmount$ | async as notificationsAmount">
-    <mat-icon *ngIf="notificationsAmount.amount" [matBadge]="notificationsAmount.amount" matBadgePosition="before"></mat-icon>
-  </span>
   <mat-menu xPosition="before" #menu="matMenu" [class]="'notification-menu'">
     <app-notifications-list
       [notificationsAmount]="notificationsAmount"
-      [recievedNotification]="recievedNotification"></app-notifications-list>
+      [receivedNotification]="receivedNotification">
+    </app-notifications-list>
   </mat-menu>
 </ng-container>
 
 <ng-template #isMobileScreen>
-  <div *ngIf="notificationsAmount$ | async as notificationsAmount" fxLayout="row" fxLayoutAlign="space-between center" class="full-width">
+  <div *ngIf="notificationsAmount$ | async as notificationsAmount"
```

Comment:
Is it possible to reuse the `notificationsAmount` property from a component to avoid creating an additional subscription?